chapters/data-processing.tex

```diff
@@ -0,0 +1,783 @@
+to correct duplicated entries is that it creates \textit{duplicates reports}
+that record which corrections made and why.
+Even if you are not using this command,
+it is important to include similar records in the \texttt{Documentation} folder.
+
+\subsection{What makes data ``tidy''}
+
+% Where these concepts come from
+Before discussing the details of how to clean your data,
+we need to define a few concepts that will be used in this chapter.
+They refer to the structure of the data,
+and have long been discussed in the database management literature.
+This section is based on Hadley Wickham's definition of tidy data,\cite{tidy-data}
+which uses a terminology that is more intuitive for users of applied statistics,
+we have added our own twist based on the jargon of development research.
+
+% Definitions
+Development data is typically acquired in the form of \textbf{tables},
+two-dimensional datasets formed by rows and columns.\sidenote{\textbf{Table:}
+	a two-dimensional dataset formed by rows and columns. 
+	Also called ``rectangular data''.
+	One example of non-rectangular data is a text.}
+Each element or cell in this table is a \textbf{data point}, that is,
+a value representing a measure of an attribute for one individual unit.\sidenote{\textbf{Data point:}
+a value representing a measure of an attribute for one individual unit.}
+For example, in a household survey,
+two of the attributes usually measured are gender and age.
+The measures of gender and age for each household member are data points.
+A \textbf{variable}\sidenote{\textbf{Variable:} the collection of all data points that measure the same attribute.}
+ is the collection of all data points that measure a given attribute.
+In the example of a household survey,
+each variable will correspond to one survey question.
+Finally, each variable will contain data points for different individuals in a given unit of observation.
+In our example, each household member is a subject, and the unit of observation is a person.
+The collection of all data points for each individual subject is an 
+\textbf{observation}.\sidenote{\textbf{Observation:} 
+	the collection of all data points representing attributes of the same individual or unit.}
+
+% Tidy data
+The ideal format to interact with any table or dataset is when
+each column represents one variable,
+each row represents one observation, 
+and all variables in a table have the same unit of observation.
+Datasets in this format are said to be \textbf{tidy},\sidenote{\textbf{Tidy data:}
+	Datasets where each column represents one variable,
+	each row represents one observation, 
+	and all variables in a table have the same unit of observation.}
+and all others are \textit{untidy}.
+This may seem trivial, but the raw data acquired is frequently untidy.
+Tidy data sets are easier to handle,
+as observations can be aggregated and linked consistently,
+and easier to clean, as each attribute only needs to be checked once.
```

Comment:
I think we need to describe what a wide data set is, as only in comparison to that, where the attribute of age is in one variable int he household member table, instead of in multiple variables where each age variable represents another household member in that household.